src/DetailsView/components/export-dialog.tsx

```diff
@@ -27,18 +29,24 @@ export interface ExportDialogDeps {
 }
 
 export const ExportDialog = NamedFC<ExportDialogProps>('ExportDialog', props => {
+    const [format, setFormat] = React.useState<ExportFormat | null>(null);
```

Comment:
@waabid @dbjorge This is the first time we'll be introducing a react hook.
I'm not against this but I do have concerns, specially the fact that this is a global dependency and how to properly test the state changes (I'm guessing the testing part is a solved problem at this point but I may be wrong).
Any thoughts?

Reply:
I don't have a particularly strong opinion on hooks vs class components as concepts, but I think it's better for maintainability to try to be consistent within a given codebase; I would lean towards using a class component instead for consistency's sake.